svc.go

```diff
@@ -137,6 +139,10 @@ func (s *SVC) Run() {
 		go func(name string, w Worker) {
 			defer s.recoverWait(name, &wg, errs)
 			if err := w.Run(); err != nil {
+				if errors.Is(err, context.Canceled) {
+					s.logger.Warn("Worker context canceled", zap.String("worker", name))
+					return
+				}
 				s.logger.Error("Run exited with error", zap.Error(err), zap.String("worker", name))
```

Comment:
I would also want to remove this line because errors are still logged after they sent to `errs` channel. What would you say about it?